nd4j/nd4j-backends/nd4j-api-parent/nd4j-native-api/src/main/java/org/nd4j/nativeblas/BaseNativeNDArrayFactory.java

```diff
@@ -78,14 +78,9 @@ public Pointer convertToNumpy(INDArray array) {
         header.capacity(size.get());
         header.position(0);
 
-        char[] magic = {'\\','x','9','3','N','U','M','P','Y','1','0'};
-
-        BytePointer magicPointer = new BytePointer(new String(magic).getBytes());
-        BytePointer bytePointer = new BytePointer(magicPointer.capacity() + (int) (size.get() + (array.data().getElementSize() * array.data().length())));
+        BytePointer bytePointer = new BytePointer((int) (size.get() + (array.data().getElementSize() * array.data().length())));
```

Comment:
Have we looked at the numpy versios to make sure the header hasn't changed?

Reply:
Yes, that wasn't the case.